carina-core/src/main/java/com/qaprosoft/carina/core/foundation/AbstractTest.java

```diff
@@ -742,11 +731,13 @@ private void generateMetadata() {
                 PrintWriter out = null;
                 try {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS2093)